src/components/ArticleInfo.jsx

```diff
@@ -0,0 +1,27 @@
+import { useParams } from 'react-router-dom';
+
+export default function ArticleInfo({articles}){
```

Comment:
There seems to be a naming mismatch here where you have called it `ArticleInfo` but imported it as `IndividualArticle`